pom.xml

```diff
@@ -132,7 +132,7 @@
                     <version>${maven-release-plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <releaseProfiles>javadoc,deploy,release,docker-push</releaseProfiles>
+                        <releaseProfiles>javadoc,deploy,release,docker-push,-console</releaseProfiles>
```

Comment:
I've checked `maven-release-plugin` documentations and:

- `release:prepare` does not has `releaseProfiles` parameters ([link](http://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html)) 
- `release:perform` has this configuration parameter ([link](http://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html)

So I think is supported but running `mvn release:prepare` will go through `kapua-console` module, while `mvn release:perform` it should not. 

Since `prepare` creates only git tag with dropped  `SNAPSHOT` version, I believe that this configuration is fine.

Regards, 

_Alberto_